dali/operators/generic/transpose/transpose.h

```diff
@@ -140,6 +69,13 @@ class Transpose : public Operator<Backend> {
     } else if (transpose_layout_ && !in_layout.empty()) {
       output_layout_ = kernels::Permute(in_layout, perm_);
     }
+  }
+
+
+  bool SetupImpl(std::vector<OutputDesc> &output_desc,
```

Comment:
Since this is only for the CPU implementation now, you could move its *cc file

Reply:
I'm using it in GPU now, followed by kernel setup.